packages/frontend/src/screens/Resources/index.js

```diff
@@ -109,7 +119,7 @@ export default function Resources() {
                   <AddDocumentIcon
                     onClick={onClickResourceHandler.bind(null, _id)}
                   />
-                  <DeleteIcon onClick={() => {}} />
+                  <DeleteIcon onClick={onDeleteclick.bind(null,_id)} />
```

Comment:
Alternatively you can use like this; `onClick={() => onDeleteclick(id)}`

Reply:
This is fixed